start_cerberus.py

```diff
@@ -79,6 +79,9 @@ def main(cfg):
         else:
             iterations = int(iterations)
 
+        kubecli.initialize_slack_client()
```

Comment:
Let's make the slack notifying feature optional to the user.

Reply:
Okay